snap/snapcraft.yaml

```diff
@@ -1,36 +1,56 @@
-name: saber
-version: 0.9.4
-summary: Handwritten notes
-description: A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and macOS!
+name: saber # you probably want to 'snapcraft register <name>'
+base: core18 # the base snap is the execution environment for this snap
+version: '0.9.4' # just for humans, typically '1.2+git' or '1.3.2'
+summary: A cross-platform libre handwritten notes app # 79 char long summary
+description: |
+  A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and the web!
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+license: GPL-3.0+
+compression: lzo
+architectures:
+  - build-on: amd64
 
-confinement: strict
-base: core18
-grade: devel
-
-slots:
-  dbus-saber:
-    interface: dbus
+plugs:
+  saber-svc:
     bus: session
+    interface: dbus
     name: com.adilhanney.saber
-
-apps:
-  saber:
-    command: saber
-    extensions: [flutter-stable]
-    plugs:
-      - network
-    slots:
-      - dbus-saber
-
 parts:
   saber:
+    # See 'snapcraft plugins'
+    plugin:  flutter
     source: .
-    plugin: flutter
-    flutter-target: lib/main_snap.dart
+    source-tag: v${SNAPCRAFT_PROJECT_VERSION}
     build-packages:
       - libsecret-1-dev
       - libjsoncpp-dev
+      - zenity
     stage-packages:
-      - libsecret-1-dev
-      - libjsoncpp1-dev
-
+      - libjsoncpp1
```

Comment:
I'm curious why you've changed this from [`flutter_secure_storage`'s instructions](https://pub.dev/packages/flutter_secure_storage#configure-linux-version)?

Reply:
They are just making self-contradiction, check the instruction properly if I'm wrong. Using development files just increases the snap size, without any use.

Reply:
I'm not an expert in this area so please can you make an issue with [flutter_secure_storage](https://github.com/mogol/flutter_secure_storage/issues) and keep the development packages for now